### PR TITLE
Minor cleanup of chapter 'Equations'

### PR DESCRIPTION
**Pull request by henrikt-ma**
Just a couple of things I stumbled upon when reading about variable initialization.

### CODE REVIEW

File: chapters/equations.tex [hunk 1]
```diff
@@ -788,77 +778,63 @@ \section{Initialization, initial equation, and initial algorithm}\doublelabel{in
...
 reduction, i.e., additional equations and, in some cases, additional unknown variables are introduced.  This whole set of equations, together with the additional constraints
 defined above, should lead to an algebraic system of equations where the number of equations and the number of all variables (including \lstinline!der! and \lstinline!pre!
 variables) is equal.  Often, this is a nonlinear system of equations and therefore it may be necessary to provide appropriate guess values (i.e., \lstinline!start! values and
-\lstinline!fixed=false!) in order to compute a solution numerically.
+\lstinline!fixed = false!) in order to compute a solution numerically.
 
-It may be difficult for a user to figure out how many initial equations have to be added, especially if the system has a higher index. A tool may add or remove initial equations
+It may be difficult for a user to figure out how many initial equations have to be added, especially if the system has a higher index.  A tool may add or remove initial equations
```
Comment (HansOlsson) [inline chapters/equations.tex:835]: This looks like adding a second space before "A"; why?

Comment (henrikt-ma) [reply chapters/equations.tex:835]: Right, that's an unrelated change purely for source code readability.  I'll remove it if you don't like it.

Comment (HansOlsson) [reply chapters/equations.tex:835]: Please do.

Comment (henrikt-ma) [reply chapters/equations.tex]: Fixed.

**Pull request merged**